src/MyPlot/provider/EconomyPlusProvider.php

```diff
@@ -21,13 +26,17 @@ public function __construct(Main $plugin)
 	 * @return bool
 	 */
 	public function reduceMoney(Player $player, float $amount) : bool {
-		if($amount < 0) {
+		if ($amount === 0) {
+			return true;
+		} elseif ($amount < 0) {
 			$amount = -$amount;
 		}
-		if($amount === 0) {
+		if($this->plugin->myMoney($player) - $amount < 0) {
+			$this->plugin->getLogger()->debug("MyPlot failed to reduce money of ".$player->getName());
```

Comment:
Formatting failed - ".$ should be " . $

Reply:
If you look at other formatting in each of the other files, the uses are intermixed

Reply:
Then run 1 formatting over all files?